src/components/calculator.js

```diff
@@ -1,41 +1,59 @@
-        <button type="button" className="operator Plus-minus operations" name="+/-">+/-</button>
-        <button type="button" className="operator" name="%">%</button>
-        <button type="button" className="operator operation" name="+">+</button>
+        <button type="button" className="operator all-clear" name="AC" onClick={this.handleClick}>AC</button>
+        <button type="button" className="operator Plus-minus operations" name="+/-" onClick={this.handleClick}>+/-</button>
+        <button type="button" className="operator" name="%" onClick={this.handleClick}>%</button>
+        <button type="button" className="operator operation" name="+" onClick={this.handleClick}>+</button>
 
-        <button type="button" className="operator number" name="7">7</button>
-        <button type="button" className="operator number" name="8">8</button>
-        <button type="button" className="operator number" name="9">9</button>
-        <button type="button" className="operator operation" name="x">&times;</button>
+        <button type="button" className="operator number" name="7" onClick={this.handleClick}>7</button>
+        <button type="button" className="operator number" name="8" onClick={this.handleClick}>8</button>
+        <button type="button" className="operator number" name="9" onClick={this.handleClick}>9</button>
+        <button type="button" className="operator operation" name="x" onClick={this.handleClick}>&times;</button>
 
-        <button type="button" className="operator number" name="4">4</button>
-        <button type="button" className="operator number" name="5">5</button>
-        <button type="button" className="operator number" name="6">6</button>
-        <button type="button" className="operator operation" name="-">-</button>
+        <button type="button" className="operator number" name="4" onClick={this.handleClick}>4</button>
+        <button type="button" className="operator number" name="5" onClick={this.handleClick}>5</button>
+        <button type="button" className="operator number" name="6" onClick={this.handleClick}>6</button>
+        <button type="button" className="operator operation" name="-" onClick={this.handleClick}>-</button>
 
-        <button type="button" className="operator number" name="1">1</button>
-        <button type="button" className="operator number" name="2">2</button>
-        <button type="button" className="operator number" name="3">3</button>
-        <button type="button" className="operator operation" name="÷">&divide;</button>
+        <button type="button" className="operator number" name="1" onClick={this.handleClick}>1</button>
+        <button type="button" className="operator number" name="2" onClick={this.handleClick}>2</button>
+        <button type="button" className="operator number" name="3" onClick={this.handleClick}>3</button>
+        <button type="button" className="operator operation" name="÷" onClick={this.handleClick}>&divide;</button>
```

Comment:
[ Optional Changes ]

- [ ] I believe, according to the design, it is supposed to be the addition sign that should be here and not the division sign. You must have interchanged them.